src/main/java/me/zeroeightsix/kami/mixin/client/MixinWorld.java

```diff
@@ -17,4 +18,17 @@ private void checkLightFor(EnumSkyBlock lightType, BlockPos pos, CallbackInfoRet
             ci.setReturnValue(false);
         }
     }
+    @Inject(method = "isRaining", at = @At("RETURN"), cancellable = true)
+    private void isRaining(CallbackInfoReturnable<Boolean> cir) {
+        if (AntiWeather.INSTANCE.isEnabled()) {
+            cir.setReturnValue(false);
+        }
+    }
+
+    @Inject(method = "getRainStrength", at = @At("RETURN"), cancellable = true)
+    private void getRainStrength(float delta, CallbackInfoReturnable<Float> cir) {
+        if (AntiWeather.INSTANCE.isEnabled()) {
+            cir.setReturnValue(0f);
+        }
+    }
```

Comment:
You forgot to make it returns false for thundering as well

Reply:
# :moyai: